apis/config/v1beta1/configuration_types.go

```diff
@@ -325,7 +325,6 @@ type Integrations struct {
 	//  - "ray.io/rayjob"
 	//  - "ray.io/raycluster"
 	//  - "jobset.x-k8s.io/jobset"
-	//  - "kubeflow.org/mxjob"
```

Comment:
Please also remove:

https://github.com/kubernetes-sigs/kueue/blob/9fa7049ec59e4d5cced9b97551a72abaad9d48b6/config/components/manager/controller_manager_config.yaml#L50

and 

https://github.com/kubernetes-sigs/kueue/blob/9fa7049ec59e4d5cced9b97551a72abaad9d48b6/charts/kueue/values.yaml#L106